app/components/notification-list/template.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+    <div class="row">
+        <div class="notification-list col-md-8">
+            <div class="notification-list-heading-container">
+                <h4 class="notification-list-heading">{{t 'components.notificationList.heading'}}</h4>
+                <div class="helper-text">
+                    {{t 'components.notificationList.helperText'}}
+                    <a href={{serviceLinks.settingsNotifications}}>
+                        {{t 'components.notificationList.userSettings'}}
```

Comment:
There is some extra space after the link here. Add a tilde to remove it: `{{t 'components.notificationList.userSettings'~}}`.

Before:
![screen shot 2018-04-04 at 9 58 45 am](https://user-images.githubusercontent.com/7131985/38313089-59f79bdc-37f1-11e8-88c9-daebb2491b33.png)

After:
![screen shot 2018-04-04 at 10 14 42 am](https://user-images.githubusercontent.com/7131985/38313098-5f9b3972-37f1-11e8-806b-0049d7e1bef8.png)


See [stack overflow](https://stackoverflow.com/questions/15953293/handlebars-whitespace-control) or [Handlebars docs](http://handlebarsjs.com/expressions.html) for more info on controlling whitespace in handlebars.